homework-19.js

```diff
@@ -0,0 +1,145 @@
+// Створити та описати сутності Багатоквартирного будинку, квартири, мешканця.
+// Додати можливість створювати нові будинки на певну кількість квартир.
+// Не обмежувати кількість мешканців у квартирі
+
+
+
+console.log('=======persons========');
+
+class Person {
+    constructor(name, surname, age, job) {
+        this.name = name;
+        this.surname = surname;
+        this.age = age;
+        this.job = job;
+    }
+
+    fullName() {
```

Comment:
```suggestion
    getFullName() {
```

ось хороший документ про неймінг https://github.com/kettanaito/naming-cheatsheet